one-ui/src/test/java/com/marklogic/hub/oneui/AbstractOneUiTest.java

```diff
@@ -124,6 +124,11 @@ protected File getFileFromClasspath(String resourceName) {
 
     protected ObjectNode readJsonObject(String json) {
         try {
+            if(objectMapper.readTree(json).isArray()) {
```

Comment:
If you need a method that returns an ArrayNode, let's not modify this - just add a "readJsonArray" method.